compiler/test/compilable/aligned2.c

```diff
@@ -0,0 +1,8 @@
+struct S
+{
+  int i;
+  short v;
+} __attribute__((aligned(2)));	// ignored as smaller than int
+
+_Static_assert(sizeof(struct S) == sizeof(int) * 2, "sizeof(S)");
+_Static_assert(_Alignof(struct S) == _Alignof(int), "_Alignof(S)");
```

Comment:
@kinke I'm erring on purging this behaviour from D.

```d
align(2) struct S { int i; short v; }
```
Declares a struct with size 6 and align 4. This is nonsense.

Reply:
In C11, it is illegal for an `_Alignas` specifier (the nearest equivalent to D `align`) to reduce alignment of a type/object.

Reply:
Yeah but as we fortunately don't have `packed` in D, we need to be able to replicate that via explicit `align` under-alignments.